client/signup/steps/email-signup-form/index.jsx

```diff
@@ -38,6 +39,11 @@ export default React.createClass( {
 
 		analytics.tracks.recordEvent( 'calypso_signup_user_step_submit', analyticsData );
 
+		// run and record before creating user
+		if ( ! this.props.signupDependencies.cartItem ) {
+			abtest( 'nuxTrampoline' );
```

Comment:
Added a check for `this.props.flowName === 'main'` to not step on other flows' toes.